thoth/adviser/boots/solved_software_environment.py

```diff
@@ -33,6 +35,7 @@
     from ..pipeline_builder import PipelineBuilderContext
 
 _LOGGER = logging.getLogger(__name__)
+_THOTH_ADVISER_DEPLOYMENT_CONFIGURED_SOLVERS = os.getenv("THOTH_ADVISER_DEPLOYMENT_CONFIGURED_SOLVERS", "")
```

Comment:
Could this be moved to the pipeline unit class? It could also simplify testing when tests can directly assign value to a class property.